rust/bridge/node/futures/src/exception.rs

```diff
@@ -0,0 +1,93 @@
+//
+// Copyright 2021 Signal Messenger, LLC.
+// SPDX-License-Identifier: AGPL-3.0-only
+//
+
+use neon::prelude::*;
+
+/// See [PersistentException]
+enum PersistentExceptionImpl {
+    Object(Root<JsObject>),
+    String(String),
+}
```

Comment:
@kjvalencik This is my 80% solution to "can't make a root of an arbitrary value", since we're mostly just using it to propagate exceptions and people rarely throw anything but objects and *maybe* strings.

Reply:
This seems pretty nice. I'll keep this in mind for future improvements. Might be able to support everything that has `ToObject` by adding a type marker.